layouts/partials/docs/inject/menu-before.html

```diff
@@ -0,0 +1,3 @@
+<p class="menu-before-reg-logo">
+  <img src="/REG-handbook/Reginald_hex.png" alt="Logo: Reginald the REG Chameleon"></p>
+</p>
```

Comment:
Is there a reason this is a pargraph block? Would a `div` be better? (Not at all a html expert 😬 )

Reply:
See the comment above.